modules/oe_theme_content_tender/config/overrides/core.entity_view_display.node.oe_tender.teaser.yml

```diff
@@ -0,0 +1,105 @@
+langcode: en
```

Comment:
This configuration won't be created on already created sites. It has to be created in post_updates.
See https://github.com/openeuropa/oe_theme/blob/EPIC-tender/modules/oe_theme_content_news/oe_theme_content_news.post_update.php. But I'm not sure that it should be overwritten if already created - some sites can have their own "teaser" view modes that will be broken after update.

Reply:
Teaser has to be overwritten even if it was already created.

Reply:
Due to Tender doesn't exist on existing sites it shouldn't be implemented.